sys/src/cmd/frontusb/lib/usb.h

```diff
@@ -344,8 +344,8 @@ struct DEp
 #define dprint if(usbdebug)fprint
 #define ddprint if(usbdebug > 1)fprint
 
-#pragma	varargck	type  "U"	Dev*
-#pragma	varargck	argpos	devctl	2
+//#pragma	varargck	type  "U"	Dev*
```

Comment:
just remove these.

Reply:
sure.  there are other cases where it's been commented out in harvey.  these should be cleaned up later.